fastlane/lanes/localization.rb

```diff
@@ -1,5 +1,5 @@
 # NOTE: When updating this list, ensure the locales having `promo_config: {…}` matches the list of locales
-# used in the `raw-screenshots` job (see `.circleci/config.yml`) by Firebase Test Lab
+# used in the `raw-screenshots` CI job for Firebase Test Lab
```

Comment:
Note: while technically, this CI job (to generate/trigger raw screenshots being built on FTL) does not exist anymore, I kept a mention of such a "CI job for FTL" around in this comment on purpose, to have it as a reminder for us when we'll work on making the screenshots work on Buildkite and FTL again in the future.